tests/test_notebooks.py

```diff
@@ -32,7 +32,9 @@ def test(self, display_html):
             with self.subTest(path=path):
                 notebook = parse_notebook(path)
                 for source, output in notebook:
-                    exec(source)  # pylint: disable=exec-used
+                    exec(  # pylint: disable=exec-used
+                        "import sys\nsys.path.append('./docs/examples')\n" + source
```

Comment:
I think it is enough to append the `sys.path` only once in `setUpClass` since `exec` is evaluated in the same environment:

```python
import sys

@classmethod
def setUpClass(cls):
    sys.append("./docs/examples")
```

Reply:
> I think it is enough to append the `sys.path` only once in `setUpClass` since `exec` is evaluated in the same environment:
> 
> ```python
> import sys
> 
> @classmethod
> def setUpClass(cls):
>     sys.append("./docs/examples")
> ```

Thanks